test1.txt

```diff
@@ -0,0 +1,8 @@
+Just some ipsm lorem stuff
```

Comment:
[On the Git mailing list](https://github.com/webstech/gggmail660ecd53-be25-6f82-d941-0be06350c754@example.com), Code Bot wrote ([reply to this](https://github.com/gitgitgadget/gitgitgadget/wiki/ReplyToThis)):

``````````email
This email is being reviewed.

   From: "Chris. Webster" <chris@webstech.net>
   
   2022-12-25 15:59:08.69
   
   test1.txt
   
   Signed-off-by: Chris. Webster <chris@webstech.net>
   ---
    test1.txt | 8 ++++++++
    1 file changed, 8 insertions(+)
    create mode 100644 test1.txt
   
   diff --git a/test1.txt b/test1.txt
   new file mode 100644
   index 0000000..5ade0ce
   --- /dev/null
   +++ b/test1.txt
   @@ -0,0 +1,8 @@
   +Just some ipsm lorem stuff
   +for a few lines to
   +make a change
   +big enough to
   +surpass diff
   +basic line
   +limits.
   +15:59:08.69  
   -- 
   webstech
   
   
``````````